syntax = "proto3";

message PaymentTransactionPayload {

    enum PayloadType {
        TYPE_UNSET = 0;
        ADD_PAYMENT = 1;
    }

    PayloadType payload_type = 1;
    Payment create_payment = 2;
}

message Payment {
    string public_key = 1;
    string id = 2;
}
